.travis.yml

```diff
@@ -0,0 +1,35 @@
+sudo: required
+services: [ docker ]
+language: perl
+perl: "5.18"
+env:
+  global:
+    - PKG_REPOTYPE=enterprise
```

Comment:
Do we target community? Maybe later on Monday?

Reply:
good catch, that is where the packages will by pushed, should be community